influxd.yaml

```
@@ -61,6 +67,76 @@ subpackages:
         - bash
         - yq
 
+  - name: ${{package.name}}-bitnami-compat
+    description: "compat package with bitnami/influxdb image"
+    dependencies:
+      runtime:
+        - bash
+        - coreutils
+        - curl
+        - findutils
+        - influxd
+        - influx
+        - grep
+        - libgcc
+        - procps
+        - sed
+        - wait-for-port # used in /opt/bitnami/scripts/libinfluxdb.sh
+    pipeline:
+      - uses: bitnami/compat
+        with:
+          image: influxdb
+          version-path: ${{vars.major-version}}/debian-12
+      - runs: |
+          mkdir -p ${{targets.subpkgdir}}/opt/bitnami/influxdb/bin
+          ln -sf /usr/bin/influxd ${{targets.subpkgdir}}/opt/bitnami/influxdb/bin/influxd
+          ln -sf /usr/bin/influx ${{targets.subpkgdir}}/opt/bitnami/influxdb/bin/influx
+
+          # copy etc to influxdb/etc.default
+          mkdir -p ${{targets.subpkgdir}}/opt/bitnami/influxdb/etc.default
+          cp -pr ${{targets.destdir}}/etc/* ${{targets.subpkgdir}}/opt/bitnami/influxdb/etc.default/
+          chmod -R u+rwX,g+rwX,o+rw ${{targets.subpkgdir}}/opt/bitnami/
+
+          mkdir -p ${{targets.subpkgdir}}/bitnami/influxdb
+          chmod -R u+rwX,g+rwX,o+rw ${{targets.subpkgdir}}/bitnami/influxdb
+
+          mkdir -p ${{targets.subpkgdir}}/docker-entrypoint-initdb.d/
+
+          # Use package path while unpacking
+          find . -iname "*.sh" -exec sed 's#/opt/bitnami#${{targets.subpkgdir}}/opt/bitnami#g' -i {} \;
+            ${{targets.subpkgdir}}/opt/bitnami/scripts/influxdb/postunpack.sh || true
```

`|| true` leads to false-positive and previously we had a customer issue for this.

```suggestion
            ${{targets.subpkgdir}}/opt/bitnami/scripts/influxdb/postunpack.sh
```

---

thank you so much! just updated my changes!